python/sanlock.c

```diff
@@ -1540,7 +1540,7 @@ Flags\n\
                             next renewal, ending this event.\n\
   SETEV_REPLACE_EVENT       Replace the existing event/data values of the\n\
                             current event. Without this flag, the operation\n\
-                            will raise SanlockException with -EBUSY error.\n\
+                            will raise sanlockException with -EBUSY error.\n\
```

Comment:
If you want to improve the docstrings, you can replace SanlockException with sanlock.SanlockException.

Reply:
will revert the sanlock exception name changes. wasn't the original intention to do that anyway.